stable/handbrake/3.0.0/questions.yaml

```diff
@@ -51,14 +69,32 @@ questions:
       $ref:
         - "definitions/timezone"
 
+  # PUID
   - variable: PUID
     group: "Container Image"
     label: "PUID"
     description: "The UserID of the user running the application and owning the files"
     schema:
       type: int
       default: 568
+  
+  #PGID
+  - variable: PGID
+    group: "Container Image"
+    label: "PGID"
+    description: "The groupID of the user/group running the application and owning the files"
+    schema:
+      type: int
+      default: 568
 
+  #UMASK
+  - variable: UMASK
+    group: "Container Image"
+    label: "UMASK (advanced)"
+    description: "The UMASK used if supported by the application"
+    schema:
+      type: string
+      default: "002"
```

Comment:
The whole section with "Container Image" variables needs to be a direct copy-pasta from something like jackett. There can't be changes because I maintain this using bulk edits.